### ISSUE TITLE
[NFR] Add ACL

### ISSUE THREAD

**Issue by ruudboon** [phalcon/phalcon-api#18]
Would it be possible to add some kind of ACL allowing resource access to certain users?

**Comment by niden** [comment]
Yes that was in the TODO list when the project was created. I am not sure if I will have enough time in the next few weeks to address this but will try.

**Comment by xaero7** [comment]
How to use firewall? or ACL?
the urls are broken:
`https://docs.phalcon.io/4.0/en/api/phalcon_firewall#firewall-adapter-acl`
